invenio_files_rest/helpers.py

```diff
@@ -166,11 +176,17 @@ def send_stream(stream, filename, size, mtime, mimetype=None, restricted=True,
     # Set cache-control
     if not restricted:
         rv.cache_control.public = True
+        # get the value of the flask config variable "SEND_FILE_MAX_AGE_DEFAULT" and use it to set how long (in
+        # seconds) a file can be cached before it is considered stale and the client should fetch the newest version.
         cache_timeout = current_app.get_send_file_max_age(filename)
+
         if cache_timeout is not None:
             rv.cache_control.max_age = cache_timeout
             rv.expires = int(time() + cache_timeout)
 
+    if restricted:
```

Comment:
Since you are changing the code because tests are failing, just add an `else` here?
```suggestion
    else:
```